Test2/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,74 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Test2.Pages
+{
+    internal class HistoryPage 
+    {
+        private IWebDriver driver;
+        public HistoryPage(IWebDriver webDriver)
+        {
+            driver = webDriver;
+        }
+        public List<IWebElement> LastResult => driver.FindElements(By.XPath("//table/tr[@class='data-td'][1]/td")).ToList();
```

Comment:
Please add a blank line and make it `private`
```suggestion
        }
        
        privatge List<IWebElement> LastResult => driver.FindElements(By.XPath("//table/tr[@class='data-td'][1]/td")).ToList();
```